setup.py

```diff
@@ -11,14 +11,16 @@
 from setuptools.command.test import test as TestCommand
 from setuptools.command.build_ext import build_ext as BuildExtCommand
 
-if sys.version_info < (3, 6):
+min_version = (3, 6)
+
+if sys.version_info < min_version:
     error = """
-Beginning with Matplotlib 3.1, Python 3.6 or above is required.
+Beginning with Matplotlib 3.1, Python {} or above is required.
```

Comment:
When did we drop 2.6? Do we need to worry about it for this message?
```suggestion
Beginning with Matplotlib 3.1, Python {0} or above is required.
```

Reply:
We have not supported 2.6 for a while, but it can not hurt.